Week2/homework/js-exercises/evenOddReporter.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+{
+  for (let i = 0; i <= 20; i++) {
+    if (i % 2 === 0) {
+      console.log(`The number ${i} is odd!`);
+    } else {
+      console.log(`The number ${i} is even!`);
+    }
+  }
+}
```

Comment:
👍 for using `use strict` and block scope. However, there is a small mistake in your program. If a number is divisible by 0, then it is even and not odd. Could you have a look please?